packages/shared/ReactVersion.js

```diff
@@ -7,4 +7,5 @@
 
 'use strict';
 
+// TODO: this is special because it gets imported during build.
```

Comment:
The release script uses this, but it would be easy to change it. Looks like the only other reference is our `version-check` task which would also be easy to update.

I think those are the only 2 things blocking this TODO? If so, I'd be happy to help with a follow-up PR.

Reply:
Maybe the cleanest way would be to shim this file with a fake module that is generated by reading `package.json`.